packages/frontend/src/components/Election/Results/IRV/winner.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+  Describe, to the default level of detail, how the IRV (Ware) tally found a
+  winner.
+*/
+
+import Typography from '@mui/material/Typography';
+import WidgetContainer from '../components/WidgetContainer';
+import Widget from '../components/Widget';
+import {
+  irvRoundResults
+} from "@equal-vote/star-vote-shared/domain_model/ITabulators";
+import ResultsBarChart from "../components/ResultsBarChart";
+import { irvContext, irvWinnerSearch } from "./ifc";
+
+const chartDataFrom = (
+  {context, round}: {
+    context: irvContext,
+    round: irvRoundResults
+  }
+) => {
+  const {candidatesByIndex} = context;
+  return round.standings.map(
+    ({ candidateIndex, hareScore }) =>
+    ({ name: candidatesByIndex[candidateIndex].name, votes: hareScore })
+  )
+};
+
+export function IRVWinnerView ( {win, context}:{
+  win: irvWinnerSearch, context: irvContext
+}) {
+  const {t} = context;
```

Comment:
nit

Instead of passing ``t`` through the functions using irvContext. You can also use ``const {t} = useRace();`` everywhere. The useRace context provider was added the the project in order to minimize how many variables we need to pass between functions.

https://react.dev/learn/passing-data-deeply-with-context